luigi_pipeline/lib/hail_tasks.py

```diff
@@ -64,10 +74,10 @@ def run(self):
         mt.write(self.output().path)
 
     def import_vcf(self):
-        # Import the VCFs from inputs.
+        # Import the VCFs from inputs. Set min partitions so that local pipeline execution takes advantage of all CPUs.
         return hl.import_vcf([vcf_file for vcf_file in self.source_paths],
                              reference_genome='GRCh' + self.genome_version,
-                             force_bgz=True)
+                             force_bgz=True, min_partitions=64)
```

Comment:
import_vcf defaults to 1 partition for .gz files.  I'm not sure where repartitioning happens in the "optimized" version of this script (which is what's being used for current dataproc runs in production), but for local, I figured 64 would be sufficient. 
I could set it to 1000, but AFAIK this causes VEP batch sizes to be no-larger than the partition size which can result in slower total VEP runtime for small VCFs. I'm planning to follow up with the hail team on the VEP issue (in general VEP in spark local mode seems to be running much slower than VEP outside hail). Until then, I'll change this to 500 unless there is a better default.

Reply:
Ah, ok. For some reason, I was thinking it would create a partition per core by default (we're using `force_bgz`, so import shouldn't be limited to one core). But looks like I was mistaken.